src/kafka-topics/list/control-topics-list.html

```diff
@@ -12,13 +12,14 @@
     <!--Topics List-->
     <div flex>
       <md-list class="md-dense" flex>
-        <md-list-item
-                dir-paginate="topic in controlTopics| orderBy:'topic' | filter : search | itemsPerPage: 7"
+        <md-list-item dir-paginate="topic in controlTopics| orderBy:'topic' | filter : search | itemsPerPage: topicsPerPage"
```

Comment:
what is the reason we want to show more topics in the left list? Will always be 7 right?